packages/fiori/src/themes/Bar.css

```diff
@@ -18,15 +18,15 @@
 }
 
 .ui5-bar-root .ui5-bar-startcontent-container {
-	padding-inline-start: 0.5rem;
+	padding-inline-start: 0.75rem;
 	display: flex;
 	flex-direction: row;
 	align-items: center;
 	justify-content: flex-start;
 }
 
 .ui5-bar-root .ui5-bar-content-container {
-	min-width: 30%;
+	min-width: 29%; /* Remaining space after removing paddings and margins */
```

Comment:
29% looks magic, and could lead to inaccuracies. Isn't better to use the percent that you know is allocated for this area - let's say 30 and remove the total of star end paddings (if they are known), for example calc(30% - 1.5rem)